packages/flutter_localizations/test/material/tabs_test.dart

```diff
@@ -0,0 +1,108 @@
+import 'dart:ui';
```

Comment:
I will put the test in material library or widget library so that you don't need to duplicate the semantics tester